### DISCUSSION TITLE
Make an option for all notes to match a single color

### DISCUSSION THREAD

**Discussion by rpc9943** [SynthstromAudible/DelugeFirmware#732] [Ideas]
If we could have an option to remove color gradient in track view / song view and only to be able to choose a single color, this wiuld be more useful for people who have specific midi channels or instruments (ex. synth green, flute yellow) so they can quickly recognize what instrument they're using. The gradient would not help with this so if we are able to switch it to single color, we can see all of the notes now reflect the instrument, and it can really help with organization. Alternative, could show the track color on the mute lights but id prefer it simply be the tracks/notes

On my novation sl mk3  keyboard i choose from color tracks and the keyboard lights up that color for the specific synth assigned, so if i could also see the notes in that tracks color match up with the keyboard it would really work well! I wouldn't be guessing what instrument is assigned, visually it would be easier to recognize.

Its a minor request but i think it would change my studio to the next level.

**Comment by n3ptun3s** [comment]
This would be super handy.
Maybe there could be even the option to highlight the 3rds, 5ths, 7ths and octaves in adjustable colors.
